src/app_service/service/contacts/service.nim

```diff
@@ -721,7 +721,8 @@ QtObject:
 
     if self.contacts.hasKey(publicKey):
       self.contacts[publicKey].dto.trustStatus = TrustStatus.Unknown
```

Comment:
should not we reset `trustStatus` inside of following if?

Reply:
This method is also used to remove the "untrustWorthy" mark. In this case, the verification status may not be Verified/Trusted